### PR TITLE
feat: update few DAs references with new values

### PR DESCRIPTION
**Pull request by akocbek**
### Description

- [update rag stack to use existing WatsonX services](https://github.com/terraform-ibm-modules/dev-rag/issues/10)
- [Pass Elastic CRN output to Sample App Config DA](https://github.com/terraform-ibm-modules/dev-rag/issues/16)
- [Watson Discovery should be disabled by default](https://github.com/terraform-ibm-modules/stack-retrieval-augmented-generation/issues/166)
- [Pass KMS instance to Watson and App Config DAs](https://github.com/terraform-ibm-modules/dev-rag/issues/13)
- [Input for existing DB instance (Elastic)](https://github.com/terraform-ibm-modules/dev-rag/issues/15)
- updated the version of DAs

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by in-1911** [comment]
@akocbek - can we include a fix for #15 in this as well?

File: stack_definition.json [hunk 1]
```diff
@@ -636,6 +695,14 @@
         {
           "name": "secrets_manager_region",
           "value": "ref:../5a - Security Service - Secret Manager/outputs/secrets_manager_region"
+        },
+        {
+          "name": "elastic_instance_crn",
+          "value": "ref:./members/3b - Databases for Elasticsearch/outputs/crn"
```
Comment (in-1911) [inline stack_definition.json]: Not sure if that's a correct reference - other inputs use `ref:../<member name>/outputs/...`

**Review by in-1911 — CHANGES_REQUESTED** [review]
Ref for elastic_instance_crn may need to be fixed. Also wonder if fixes for #15 can be added.

**Comment by akocbek** [comment]
added [elastic_instance_crn](https://github.com/terraform-ibm-modules/dev-rag/issues/15)

**Review by in-1911 — DISMISSED** [review]

File: stack_definition.json [hunk 2]
```diff
@@ -717,35 +717,35 @@
   "outputs": [
     {
       "name": "elasticsearch_hostname",
-      "value": "ref:./members/3b - Databases for Elasticsearch/outputs/hostname"
+      "value": "ref:../3b - Databases for Elasticsearch/outputs/hostname"
     },
     {
       "name": "elasticsearch_port",
-      "value": "ref:./members/3b - Databases for Elasticsearch/outputs/port"
+      "value": " ref:../3b - Databases for Elasticsearch/outputs/port"
```
Comment (in-1911) [inline stack_definition.json]: @akocbek - I think you have an extra leading space

Comment (akocbek) [reply stack_definition.json]: @in-1911 good catch, thanks

**Review by in-1911 — APPROVED** [review]

**Pull request merged**